src/org/omegat/util/StaticUtils.java

```diff
@@ -83,11 +86,26 @@ private StaticUtils() {
      * Configuration directory on Mac OS X
      */
     private static final String OSX_CONFIG_DIR = "/Library/Preferences/OmegaT/";
+    /**
+     * Application data directory on Windows platforms.
+     */
+    private static final String WINDOWS_LOCAL_DATA_DIR = "AppData\\Local";
+    private static final String WINDOWS_DATA_DIR = "\\OmegaT\\";
+
+    /**
+     * Application data directory on UNIX platforms.
+     */
+    private static final String UNIX_DATA_DIR = "/.local/share/OmegaT/";
+
+    /**
+     * Application data directory on macOS.
+     */
+    private static final  String OSX_DATA_DIR = "/Library/Application Support/OmegaT/";
 
     /**
      * Script directory
      */
-    private static final String SCRIPT_DIR = "script";
+    private static final String SCRIPT_DIR = "scripts";
```

Comment:
```suggestion
    private static final String SCRIPT_DIR = "script";
```

This should be "script" for backward compatibility